REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
Yep. This PR is not going to be compatible with 0.4 AFAICT. To make it compatible, it would need all the guard and/or Compat code added, at which point `julia 0.4` can be put back into REQUIRE. My personal preference is to make this next tagged version (`0.9.0`) only compatible with Julia 0.5-. Users can choose to restrict TimeSeries to `0.8.8` in their code if it uses Julia 0.4. I don't foresee that much more functionality coming into the package, though there is always room to improve performance.